docs/manuals/source/DeveloperGuide/ReleasingBareos.rst

```diff
@@ -0,0 +1,77 @@
+Releasing Bareos
+================
+
+This chapter describes how to release a new version of Bareos.
+The documentation is limited to the changes that need to be done to the sources and the git repository.
+Building and distributing source and binary packages is not the scope of this chapter.
+
+.. note::
+
+   Consider this a step-by-step manual. The sections are meant to be carried out in that order.
+
+Deciding for a version number
+-----------------------------
+When you release you will usually need at least two version numbers: the version you're going to release and the version that will follow after that version.
+For details on the numbering-scheme, take a look at :ref:`section-version-numbers`.
+
+Usually, you will have an existing work-in-progress tag that generates pre-release version numbers for your branch.
+So the version you're going to release is probably the final version for that pre-release.
+
+Preparing the release notes
+---------------------------
+For each new release there should be a section in the :ref:`releasenotes`.
+You can use the release notes of a previous release as a template.
```

Comment:
See :ref:`DocumentationStyleGuide/BareosSpecificFormatting/Release.html:release-notes` about formatting the release notes.